app/javascript/controllers/map_controller.js

```diff
@@ -157,18 +148,16 @@ export default class extends Controller {
 
   setMapZoom(zoom, amount, width, height) {
     this.imageTarget.dataset.zoom = zoom
-    this.imageTarget.dataset.zoomAmount = parseFloat(amount)
     this.imageTarget.dataset.width = width
     this.imageTarget.dataset.height = height
-    this.imageTarget.style.backgroundSize = `${this.imageTarget.dataset.width}px ${this.imageTarget.dataset.height}px`
+    this.imageTarget.dataset.zoomAmount = amount
+    this.disableZoomButtons()
```

Comment:
What do you think about naming this s one thing like `updateZoomButtons` since it can also enable them?

Reply:
Renamed in 807b4f4